doc/modules/model_evaluation.rst

```diff
@@ -890,15 +890,15 @@ In this context, we can define the notions of precision, recall and F-measure:
 
 .. math::
 
-   \text{precision} = \frac{tp}{tp + fp},
+   \text{precision} = \frac{\text{tp}}{\text{tp} + \text{fp}},
 
 .. math::
 
-   \text{recall} = \frac{tp}{tp + fn},
+   \text{recall} = \frac{tp}{\text{tp} + \text{fn}},
 
 .. math::
 
-   F_\beta = (1 + \beta^2) \frac{\text{precision} \times \text{recall}}{\beta^2 \text{precision} + \text{recall}}.
+   F_\beta = \frac{(1 + \beta^2) \text{tp}}{(1 + \beta^2) \text{tp} + \text{fp} + \beta^2 \text{fn}}.
 
```

Comment:
I think it's still worth noting the equivalence to the other formula, but just note that it's defined as 0 when TP is 0. The user guide is the place to put that level of detail.

Reply:
I'm confused what you mean by:

> The user guide is the place to put that level of detail.

This section corresponds to [section 3.3.2.9](https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures) of the user guide

Reply:
Yes, I mean that you provided that level of detail - describing the relationship between two formulations - in the clas docstring, when it instead belongs here.